baselines/tamuna/tamuna/conf/base.yaml

```diff
@@ -0,0 +1,35 @@
+meta:
+  n_repeats: 3
+  seed: 42
+
+dataset:
+  iid: True
+
+model:
+  _target_: models.Net
+  num_classes: 10
+
+strategy:
+  _target_: strategy.TamunaStrategy
+
+server:
+  clients_per_round: 10
+  num_clients: 1000
+  num_rounds: 35
+  # sparsity parameter, 2 <= s <= clients_per_round
+  s: 4
+  # probability of server communication, i.e. clients will in expectation do 1 / p local epochs
+  # number of local epochs each round is synchronized across the clients
+  p: 0.333
+  server_device: 'cuda:0'
+  uplink_factor: 1
+  downlink_factor: 1
+
+client:
+  client_device: 'cuda:0'
+  client_resources:
+    num_cpus: 4
+    num_gpus: 1
```

Comment:
Since your model is fairly small, i'm wondering if it would be better to use a smaller `num_gpus`. For instance setting it to 0.2 would allow for 5 concurrent clients on a GPU (total <5GB of VRAM usage when running the exp.)

Reply:
I changed the resources to 2 cpus and 0.2 gpu.